app/frontend/css/cdo-task-list.scss

```diff
@@ -0,0 +1,56 @@
+.defra {
+
+  &-summary-width {
+    width: 37.5% !important;
+  }
+
+  &-actions-header {
```

Comment:
Have you thought about the mobile styling of this (although I guess for Portal users maybe not an issue)